src/components/Popup/Popup.scss

```diff
@@ -6,6 +6,12 @@
     -o-animation: fadein 0.5s; /* Opera < 12.1 */
     animation: fadein 0.5s;
   }
+
+  .tm-popup-close-bt {
+    height: 40px !important;
+    width: 40px !important;
```

Comment:
hmm could we avoid '!important' there ?

Reply:
If we define it above, but I thought it clearer to define it in Popus.scss, and add important to avoid to be rewritten.

Reply:
@edellucien more explicit rules overwrite less explicit ones.
`!important` is almost never needed.